src/home/index.js

```diff
@@ -0,0 +1,12 @@
+import './make-user.js';
```

Comment:
Finish importing the makeUser function here might look like 
`import makeUser from './make-user.js';`